.github/workflows/ci.yml

```
@@ -26,10 +26,8 @@ jobs:
     - name: Initialize Pants
       uses: pantsbuild/actions/init-pants@d64f38147540d5607bfb8d812f95d5285a8cff3d  # v8
       with:
-        gha-cache-key: v0
+        gha-cache-key: "${{ runner.os }}"
```

```suggestion
        gha-cache-key: "v0-${{ runner.os }}"
```

its helpful to have a bustable cache part